tests/integration/archive_test.go

```diff
@@ -77,6 +77,8 @@ func TestArchive_StableDigest(t *testing.T) {
 
 	// Use a fixed bundle to work with so that we can rely on the registry and layer digests
 	const reference = "ghcr.io/getporter/examples/whalegap:v0.2.0"
+	const referencedImg = "carolynvs/whalesayd@sha256:8b92b7269f59e3ed824e811a1ff1ee64f0d44c0218efefada57a4bebc2d7ef6f"
```

Comment:
I was expecting that these extra validations would be added to the Airgapped test above, since this feature supports airgap and testing it requires that we are making it impossible to use the original images by taking the original registry offline.

The test you are editing is for checking that when we archive twice, we get the same digest.

Can you move these checks into the other test?

Reply:
Sounds good